Backend/src/main/java/com/login/demologin/controller/UserController.java

```diff
@@ -12,41 +14,45 @@
 
 @CrossOrigin(origins = "http://localhost:4200")
 @RestController
-@RequestMapping("/demo")
-public class MainController {
+@RequestMapping("/User")
+public class UserController {
     @Autowired
     private UserRespository userRespository;
-    @PostMapping("/create")
+
+    @PostMapping("/CreateUser")
```

Comment:
I believe this can be replaced by `@PostMapping("/")`, according to the REST standards for URLs

Reply:
yes ..... probably It's the best way .... I just check out the documents of REST standards for URLs